experimental/torch_xla2/test/test_ops.py

```diff
@@ -213,6 +212,7 @@ def run_export_and_compare(testcase,
         sample_input.input, sample_input.args, sample_input.kwargs))
       with testcase.env:
         res2 = func(input2, *args2, **kwargs2)
+        print("input2: ", input2, "sample_input", sample_input.input, " res2:  ", res2)
```

Comment:
thanks, do we want to keep these `print` code here?

Reply:
removed.